src/main/java/com/flipkart/databuilderframework/engine/DataBuilderExecutionListener.java

```diff
@@ -28,4 +28,8 @@ void afterException(
             DataDelta dataDelta,
             Map<String, Data> prevResponses,
             Throwable frameworkException) throws Exception;
+
+    void postProcessing(DataFlow dataFlow,
+                        DataDelta dataDelta,
+                        Map<String, Data> allResponses) throws Exception;
```

Comment:
Pass in the full execution response. There might be something else getting added. That will also get passed in automatically.

Reply:
Pass in a boolean denoting if there was an error or not. Better still, if you can actually pass in the error object that is thrown.